### DISCUSSION TITLE
Saludo

### DISCUSSION THREAD

**Discussion by JoselinMiel** [Launch-X-Latam/MisionFrontEnd#478] [Team Posts]
Saludo Para todos

**Comment by ciberelu** [comment]
Como saben que ya están en backend ?

**Comment by themau1111** [comment]
dejen de contestar, estan enviando muchos correos, las instrucciones ya estan en teams y github
https://github.com/Launch-X-Latam/OnBoarding/tree/main/join_github_teams

**Comment by Josephr1318** [comment]
El correo de one boarding de bienvenida lleva un calendario de guia
